Configuration/RealmSwift/Base.xcconfig

```diff
@@ -28,13 +28,16 @@ SWIFT_WHOLE_MODULE_OPTIMIZATION = YES;
 MACOSX_DEPLOYMENT_TARGET = 10.9;
 IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 WATCHOS_DEPLOYMENT_TARGET = 2.0;
+TVOS_DEPLOYMENT_TARGET = 9.0;
 CODE_SIGN_IDENTITY[sdk=iphone*] = iPhone Developer;
 TARGETED_DEVICE_FAMILY = 1,2,4;
```

Comment:
I believe you want to add `3` to this as well. I've needed to in order to get everything to behave with other frameworks.